ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
+    # OIDC_PROVIDER = "https://accounts.google.com"
+    # OIDC_CLIENT_ID = "XXXX"
+    # OIDC_CLIENT_SECRET = "XXXX"
+    SERVER_NAME = "localhost:8001"
```

Comment:
SERVER_NAME is a difficult one. Flask can't know which DNS name (interface) it's supposed to listen to on multi-target hosts unless you tell it, hence SERVER_NAME. I can try and do a default version but allow override with SERVER_NAME